homeassistant/components/homewizard_energy/sensor.py

```diff
@@ -0,0 +1,217 @@
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_ENERGY_EXPORT_T1_KWH,
+        name="Total Power Export T1",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_ENERGY_EXPORT_T2_KWH,
+        name="Total Power Export T2",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_W,
+        name="Active Power",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_L1_W,
+        name="Active Power L1",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_L2_W,
+        name="Active Power L2",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_ACTIVE_POWER_L3_W,
+        name="Active Power L3",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=ATTR_TOTAL_GAS_M3,
+        name="Total Gas",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        device_class=DEVICE_CLASS_GAS,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key=ATTR_GAS_TIMESTAMP,
+        name="Gas Timestamp",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+    ),
+)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Config entry example."""
+    energy_api = hass.data[DOMAIN][entry.unique_id][CONF_API]
+    coordinator = hass.data[DOMAIN][entry.unique_id][COORDINATOR]
+
+    entities = []
+    for description in SENSORS:
+        if description.key in energy_api.data.available_datapoints:
+            entities.append(HWEnergySensor(coordinator, entry, description))
+    async_add_entities(entities)
+
+
+class HWEnergySensor(CoordinatorEntity, SensorEntity):
+    """Representation of a HomeWizard Energy Sensor."""
+
+    unique_id = None
+    name = None
+
+    def __init__(self, coordinator, entry, description):
+        """Initialize Sensor Domain."""
+
+        super().__init__(coordinator)
+        self.entity_description = description
+        self.entry_data = entry.data
+
+        # Config attributes.
+        self.name = "{} {}".format(entry.data["name"], description.name)
```

Comment:
Use string formatting.
```suggestion
        self._attr_name = f"{entry.data['name']} {description.name}"
```